lib/std/os/linux.zig

```diff
@@ -815,7 +809,12 @@ pub fn fchown(fd: i32, owner: uid_t, group: gid_t) usize {
 }
 
 pub fn fchmodat(fd: i32, path: [*:0]const u8, mode: mode_t, flags: u32) usize {
```

Comment:
Can you drop the "flags" argument to this `fchmodat` since its ignored?  I think the Zig wrapper here should reflect what the underlying syscall expects.